plugins/kubernetes/app/helpers/deploy_group_roles_helper.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 # NOTE: using kubernetes namespace resulted in a strange error on every page
 module DeployGroupRolesHelper
+  def show_istio_sidecar_ui?
+    Samson::EnvCheck.set?('ISTIO_INJECTION_SUPPORTED')
```

Comment:
I suggest to extract constant ISTIO_INJECTION_SUPPORTED

Reply:
constants make it harder to test both enabled and disabled cases